opentok/opentok-meet
# OPENTOK-47982: Add updated e2ee api

#### What is this PR doing?
- Update usage of e2ee API.
- Use custom version of opentok-angular to support connection options


#### How should this be manually tested?
- checkout branch and run npm install
- run locally
- create a new room, enabling e2ee with a secret longer than 8 characters
- join room with same secret from 2nd tab, ensure all works ok
- join with different secret, check it doesn't work
- join with e2ee false and check it doesn't work

#### What are the relevant tickets?
OPENTOK-47982
